### PR TITLE
New inside-rust blog post -- The Rustc Learning Working Group - An Introduction

### PR DESCRIPTION
**Pull request by amanjeev**
Hi, I wrote a blog post, as promised a while ago, about the Learning Working Group. Please let me know if you see it a good fit for the blog or needs improvements. Thank you.

The promise of blog post was made here https://rust-lang.zulipchat.com/#narrow/stream/196385-t-compiler.2Fwg-learning/topic/blog/near/177905642

### CODE REVIEW

**Comment by spastorino** [comment]
@amanjeev this is great :heart: :heart: :heart:. Just loudly thinking ... maybe we can make a more direct invitation for people to get involved at the end, something like, if you're interested in this effort, please join us on Zulip, introduce yourself, let us know what do you find interesting about the effort and we can collaborate make rustc-guide better. Something maybe along those lines but better written maybe?.

**Comment by amanjeev** [comment]
@spastorino I added a small section for call for participation, inspired by your points above. Please check.

File: posts/inside-rust/2019-10-26-rustc-learning-working-group-introduction.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+We are always in need of help from both existing compiler contributors
+and new folks who want to contribute and learn.
+
+## Call for participation
+
+There is no bar to entry in the group. To join the group, you can
+drop a message in [`#t-compiler/wg-learning` on Zulip] introducing
+yourself and maybe if you have ideas on improvements to rustc-guide.
+We would love to hear from you. Hope you will reach out and help improve
+the documentation.
```
Comment (spastorino) [inline posts/inside-rust/2019-10-26-rustc-learning-working-group-introduction.md]: What about ...

```
There is no bar to entry in the group. To join the group, you can
drop a message in [`#t-compiler/wg-learning` on Zulip] introducing
yourself. We would love to hear from you and hope we all together
can make the documentation better.
```

**Comment by amanjeev** [comment]
@spastorino Made the changes per your suggestion. Please let me know if I missed something.

File: posts/inside-rust/2019-10-26-rustc-learning-working-group-introduction.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: "The Rustc Learning Working Group - An Introduction"
+author: Amanjeev Sethi
+description: "introduction learning working group useful links"
+team: the compiler team <https://www.rust-lang.org/governance/teams/compiler>
```
[the remark this answers is not in the document]
Comment (amanjeev) [reply posts/inside-rust/2019-10-26-rustc-learning-working-group-introduction.md]: I don’t think there is an anchor on that page. If you can give me the link you want to point to I can change this.

**Comment by amanjeev** [comment]
Please let me know how we merge this.

**Comment by spastorino** [comment]
> Please let me know how we merge this.

IMO is ready, I guess we just need @nikomatsakis to pass around this PR and he will merge it. I know Niko is very busy but he will come around at some point and do so :).

File: posts/inside-rust/2019-10-26-rustc-learning-working-group-introduction.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pietroalbini) [reply posts/inside-rust/2019-10-26-rustc-learning-working-group-introduction.md]: https://www.rust-lang.org/governance/teams/compiler#wg-learning

[the remark this answers is not in the document]
Comment (amanjeev) [reply posts/inside-rust/2019-10-26-rustc-learning-working-group-introduction.md]: Thank you @pietroalbini. I made the change.

File: posts/inside-rust/2019-10-26-rustc-learning-working-group-introduction.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: "The Rustc Learning Working Group - An Introduction"
+author: Amanjeev Sethi
+description: "introduction learning working group useful links"
+team: the learning working group <https://www.rust-lang.org/governance/teams/compiler#wg-learning>
```
Comment (nikomatsakis) [inline posts/inside-rust/2019-10-26-rustc-learning-working-group-introduction.md]:
```suggestion
team: the rustc learning working group <https://www.rust-lang.org/governance/teams/compiler#wg-learning>
```

(I wonder if we should rename this -- it sounds so much more general than it is)

**Pull request merged**